contrib/dokku-installer.rb

```diff
@@ -3,21 +3,26 @@
 require "open3"
 
 if ARGV[0] == "onboot"
-  File.open("/etc/init/dokku-installer.conf", "w") do |f|
-    f.puts "start on runlevel [2345]"
-    f.puts "exec #{File.absolute_path(__FILE__)} selfdestruct"
-  end if File.directory?("/etc/init")
-  File.open("/etc/systemd/system/dokku-installer.service", "w") do |f|
-    f.puts "[Unit]"
-    f.puts "Description=Dokku web-installer"
-    f.puts ""
-    f.puts "[Service]"
-    f.puts "ExecStart=#{File.absolute_path(__FILE__)} selfdestruct"
-    f.puts ""
-    f.puts "[Install]"
-    f.puts "WantedBy=multi-user.target"
-    f.puts "WantedBy=graphical.target"
-  end if File.directory?("/etc/systemd/system")
+  if File.realpath("/sbin/init") != "/sbin/init" then
```

Comment:
I don't want to merge this because the check on the path here changes how dokku works for ubuntu (you are switching to this new rbinit file) and it's unclear to me how that is better than the status quo.